docs/application/tizen-studio-0/setup/test.md

```diff
@@ -0,0 +1,135 @@
+# Grid layout(Tizen 6.0 or late)
```

Comment:
Please add a space between word "layout" and the first parenthesis.
Also, change the word "late" as "later".